compiler/infra/Cfg.hpp

```diff
@@ -34,8 +34,8 @@ class CFG : public OMR::CFGConnector
    {
    public:
 
-   CFG(TR::Compilation *comp, TR::ResolvedMethodSymbol *method) :
-      OMR::CFGConnector(comp, method) {}
+   CFG(TR::Compilation *comp, TR::ResolvedMethodSymbol *method, TR::Region *region = NULL) :
```

Comment:
Rather than adding a `TR::Region *` parameter why not just create another constructor overload that takes the region as a reference. The version without a region will default to the heap region and the other will default to the region supplied. There is just a coding convention in the compiler that we pass regions by reference precisely because NULL regions make no sense...